manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "wish-list",
   "vendor": "vtex",
-  "version": "1.7.6",
+  "version": "1.7.7",
```

Comment:
The version will be automatically updated when the PR is merged.

Reply:
Do you want me to leave the version as it was?

Reply:
Yes. leave it at 1.7.6

Reply:
Update the version